### PR TITLE
Completed Competitve coding 6 problems - please review!

### PR DESCRIPTION
**Pull request by aakanksha13**

### CODE REVIEW

File: beatifulArranement.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#Time Complexity : O(p) where p is the number of permutations which are valid
```
Comment (Himanshu-Agarwal93) [inline beatifulArranement.py:1]: You don't know the value of p. Time complexity should be exponential

**Review by Himanshu-Agarwal93 — COMMENTED** [review]
Good work. Please check the comment

**Comment by Himanshu-Agarwal93** [comment]
Good work. Please check the comment

File: beatifulArranement.py (hunk 1 above)
Comment (aakanksha13) [reply beatifulArranement.py:1]: Alright, will take care of that and make the changes.

**Pull request closed without merging**